lib/bundle/commands/check.rb

```diff
@@ -52,5 +52,13 @@ def self.any_apps_to_install?
       current_apps = Bundle::MacAppStoreDumper.app_ids
       (requested_apps - current_apps).any?
     end
+
+    def self.any_formulae_to_start?
+      @dsl ||= Bundle::Dsl.new(Bundle.brewfile)
+      @dsl.entries.select { |e| e.type == :brew }.any? do |e|
+        formula = Bundle::BrewInstaller.new(e.name, e.options)
+        !(formula.restart_service? && Bundle::BrewServices.started?(e.name))
```

Comment:
Right now Formula doesn't have option `start_service` in dsl. Do you want me to add it too?